KsApi/Service.swift

```diff
@@ -184,6 +184,10 @@ public struct Service: ServiceType {
     return request(.friendStats)
   }
 
+  public func fetchGraph<A: Swift.Decodable>(query: NonEmptySet<Query>) -> SignalProducer<A, GraphError> {
```

Comment:
Any reason to not just expose `fetch(_:)` directly?

Reply:
In this case I was just following the structure of the current Service.swift file. It may be a good idea to move the graph related functions to a separated file (I think that's how it is in our private repo). What do you think?

Reply:
Yeah I had separated them out into two different services before and kept the shared functionality in `ServiceType`, I felt like that made it easier to work with but it's up to you! The last stuff that I worked on related to that is in this branch https://github.com/kickstarter/ios-private/tree/subs-pledge

Reply:
Yeah I had separated them out into two different services before and kept the shared functionality in ServiceType, I felt like that made it easier to work with but it's up to you! Will send you the link on Slack 👍